conn2res/plotting.py

```diff
@@ -212,33 +224,34 @@ def plot_iodata(
         for tf in end_points:
             ax.plot(
                 tf * np.ones((2)), np.array([min_y, max_y]), c='black',
-                linestyle='--', linewidth=1.0
+                linestyle='--'
             )
 
     # set title
-    if title is not None:
-        plt.title(title, fontsize=12)
+    # if title is not None:
+    #     fig.suptitle(title)
```

Comment:
I suggest to uncomment this and delete lines 213-214 to set the title at the Figure level

Reply:
Actually it might be easier if I do this and can just update the pull request. Is that ok?